tracer/build_in_docker.sh

```diff
@@ -19,6 +19,7 @@ docker run -it --rm \
     --env NugetPackageDirectory=/project/packages \
     --env artifacts=/project/tracer/bin/artifacts \
     --env DD_INSTRUMENTATION_TELEMETRY_ENABLED=0 \
+    --env NUKE_TELEMETRY_OPTOUT=0 \
```

Comment:
(dupe comment) We _don't_ want to opt-out of Nuke telemetry here? You are using `=1` in the `build.[cmd|ps1|sh]` scripts.